source/geometry/IntBox2D.ooc

```diff
@@ -102,6 +102,8 @@ IntBox2D: cover {
 		this left <= point x && point x < this right && this top <= point y && point y < this bottom
 	}
 	contains: func ~box (box: This) -> Bool { this intersection(box) == box }
+	toFloatBox2D: func -> FloatBox2D { FloatBox2D new(this left, this top, this width, this height) }
```

Comment:
You can probably do that, yeah... never overloaded `as` myself, but i've seen it around.